resources/styles/components/learning-guide.less

```diff
@@ -256,144 +295,45 @@
-  }
-}
-@media screen and (min-width: 1120px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 318px !important;
-  }
-}
-@media screen and (min-width: 1140px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 324px !important;
-  }
-}
-@media screen and (min-width: 1160px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 330px !important;
-  }
-}
-@media screen and (min-width: 1180px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 336px !important;
-  }
-}
-@media screen and (min-width: 1200px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 342px !important;
-  }
-}
-@media screen and (min-width: 1220px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 348px !important;
-  }
-}
-@media screen and (min-width: 1240px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 354px !important;
-  }
-}
-@media screen and (min-width: 1260px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 360px !important;
-  }
-}
-@media screen and (min-width: 1280px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 366px !important;
-  }
-}
-@media screen and (min-width: 1300px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 372px !important;
-  }
-}
-@media screen and (min-width: 1320px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 378px !important;
-  }
-}
-@media screen and (min-width: 1340px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 384px !important;
-  }
-}
-@media screen and (min-width: 1360px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 390px !important;
-  }
-}
-@media screen and (min-width: 1380px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 396px !important;
-  }
-}
-@media screen and (min-width: 1400px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 402px !important;
-  }
-}
-@media screen and (min-width: 1420px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 408px !important;
-  }
-}
-@media screen and (min-width: 1440px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 414px !important;
-  }
-}
-@media screen and (min-width: 1460px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 420px !important;
-  }
-}
-@media screen and (min-width: 1480px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 426px !important;
-  }
-}
-@media screen and (min-width: 1500px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 432px !important;
-  }
-}
-@media screen and (min-width: 1520px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 438px !important;
-  }
-}
-@media screen and (min-width: 1540px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 444px !important;
-  }
-}
-@media screen and (min-width: 1560px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 450px !important;
-  }
-}
-@media screen and (min-width: 1580px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 456px !important;
-  }
-}
-@media screen and (min-width: 1600px) {
-  _:-ms-lang(x), .learning-guide { 
-    height: 462px !important;
-  }
-}
+.ie-resize-fix(306px, 0px);
+.ie-resize-fix(306px, 1079px);
```

Comment:
@darrenmason Would a for-loop work here?

@Fredasaurus Do these breakpoints need to be so finegrained?